sdk/storage/azblob/zc_response_helpers.go

```diff
@@ -50,11 +50,14 @@ type DownloadResponse struct {
 // while reading, it will make additional requests to reestablish a connection and
 // continue reading. Specifying a RetryReaderOption's with MaxRetryRequests set to 0
 // (the default), returns the original response body and no retries will be performed.
```

Comment:
Should this comment include something about passing in 'nil' to accept the default options?

Reply:
Yes it should, added!